lib/galaxy/model/__init__.py

```diff
@@ -8260,7 +8257,7 @@ def active(self):
         states = WorkflowInvocation.states
         return self.state in [states.NEW, states.READY]
 
-    def set_state(self, state: "WorkflowInvocation.states"):
+    def set_state(self, state: "InvocationState"):
```

Comment:
The enum class states got moved and turned into an attribute, because of this pydantic complained about typing for the set_state method. 
Is it right to change the input of the method like this?

Reply:
I guess so, but you can remove the double quotes, `InvocationState` is imported in this file.
```suggestion
    def set_state(self, state: InvocationState):
```